### ISSUE TITLE
reorganize-docs branch status

### ISSUE THREAD

**Issue by nicoddemus** [pytest-dev/pytest#4120]
https://github.com/pytest-dev/pytest/commits/reorganize-docs is the branch that @obestwalter, @pfctdayelise, and @hackebrot worked during our 2016 sprint. This has been sitting there for two years now, and unfortunately may cause confusion because the docs are online and outdated (https://github.com/pytest-dev/pytest/issues/4115#issuecomment-429194220).

@obestwalter, @pfctdayelise, and @hackebrot, would you like to keep the branch or should we delete it? I'm not sure how useful the branch is, given that it is probably riddled with conflicts by now...

**Comment by blueyed** [comment]
> given that it is probably riddled with conflicts by now...

Just one when merging master:

> CONFLICT (content): Merge conflict in AUTHORS

**Comment by nicoddemus** [comment]
Oh I'm surprised, hehehe!

**Comment by Zac-HD** [comment]
From a quick skim over the diff, it's mostly placeholder pages for the reorganization and the completed pages all cover content that has since been expanded (marks, xfail, skipping).  I'd delete the branch.

**Comment by hackebrot** [comment]
I copied the branch to my fork. Go ahead and delete it @nicoddemus! 🗑

**Comment by nicoddemus** [comment]
Deleted then! As long as it is saved somewhere we can always get it back if we want to.

Thanks everyone! 🤗

**Comment by kini** [comment]
I found some pytest documentation pages on google which had "reorganize-docs" in their URLs (e.g. https://docs.pytest.org/en/reorganize-docs/new-docs/user/directory_structure.html). I couldn't find any explanation internal to the pages themselves as to why they exist, whether they're relevant / up to date, etc., so I went searching on github, which led me to this ticket.

Do those pages come from the branch being discussed in this github issue? If so, since the branch has been deleted, should the pages also be removed from the website?
